compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/TypeChecker.java

```diff
@@ -6897,11 +6896,16 @@ private BType checkInvocationParam(BLangInvocation iExpr, AnalyzerData data) {
                             incRecordAllowAdditionalFields = false;
                         }
                         iExpr.requiredArgs.add(expr);
+                    } else if (invokableSymbol.restParam != null &&
+                            invokableSymbol.restParam.name.value.equals(((BLangNamedArgsExpression) expr).name.value)) {
+                        // can not provide a rest parameters as named args
```

Comment:
since we can see the `DiagnosticErrorCode.REST_ARG_CANNOT_BE_NAMED_ARG` being logged in the vicinity, this comment gives kind of redundant information.